octobot_trading/personal_data/orders/orders_manager.py

```diff
@@ -157,6 +157,15 @@ def _get_pending_order(self, created_order, should_pop):
                 return pending_order
         return None
 
+    def get_all_active_and_pending_orders_shared_signal_order_id(self) -> list:
+        return [
```

Comment:
Could we use _select_order instead of adding a new orders.values() iteration method ?